R/Renderer.R

```diff
@@ -191,7 +192,39 @@ Renderer <- R6::R6Class( # nolint: object_name_linter.
       args_nams <- unique(names(args))
       args <- lapply(args_nams, function(x) args[[x]])
       names(args) <- args_nams
-      do.call(rmarkdown::render, args)
+      rmd_path <- do.call(rmarkdown::render, args) # PATH IS RETURNED
+      clean_chunks <- function(input_rmd) {
```

Comment:
instead of cleaning chunks, the chunks should be an original text with the class `code_chunk` and depending on the output type it should be either ignored or transformed.